application/controllers/spritecomics/gallery.php

```diff
@@ -137,6 +138,19 @@ public function get_details($id_document = null)
 				}
 				
 				$tpl_tags	=	\Eliya\Tpl::get('spritecomics/gallery/tags', ['tags' => $tags]);
+				$imagePath = $this->request->getBaseURL() . $document->getPath();
+				$URL = $this->request->getBaseURL() . 'spritecomics/gallery/details/' . $id_document;
+				
+				\Eliya\Tpl::set([
+					'social_NW_meta'	=>	'<meta property="og:title" content="' . $page_title . '">
```

Comment:
Bon, en effet, je viens de tester et il n'y a aucun problème de ce côté là. Par contre, je viens de remarquer que si on recherche une apostrophe seule (donc pas dans une suite de caractère, vraiment seulement une apostrophe) dans la barre de recherche, on a ça :
EntityRequest::executeSQL() : An error occurs while running the given SQL request. [SQLSTATE: 42000][DriverCode: 1064] => You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '%' OR t.name IN (''')' at line 5

Bon, après c'est pas comme si tout le monde allait se mettre à rechercher ' sur le site.